AirSim/PythonClient/ros/car_image_raw.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# Example ROS node for publishing AirSim images.
+
+import setup_path 
+import airsim
+import time
+import rospy
+
+# ROS Image message
+from sensor_msgs.msg import Image
+
+def airpub():
+    pub = rospy.Publisher("airsim/image_raw", Image, queue_size=1)
+    rospy.init_node('image_raw', anonymous=True)
+    rate = rospy.Rate(15)# connect to the AirSim simulator 
+    client = airsim.CarClient()
+    client.confirmConnection()
+
+    while not rospy.is_shutdown():
+         # get camera images from the car
+        start = time.time()
+        responses = client.simGetImages([
+            airsim.ImageRequest("front_center_custom", airsim.ImageType.Scene, False, False),
+            airsim.ImageRequest("front_left_custom", airsim.ImageType.Scene, False, False)
+            ])
+        print(time.time()-start)
```

Comment:
have we added this? this seems a latency check which would make sense

Reply:
Yep